File: javascript/006/006.js
"use strict";

let sumOfSquares = 0;

let sum = 0;

for (let i = 1; i <= 100; i += 1) {
  sumOfSquares += (i * i);

  sum += i;
}

let squareOfSum = sum * sum;

console.log(squareOfSum - sumOfSquares);

File: go/006/006.go
package main

import "fmt"

func main() {
	sumOfSquares := uint64(0)

	sum := uint64(0)

	squareOfSum := uint64(0)

	for i := 1; i <= 100; i++ {
		sumOfSquares += uint64(i * i)

		sum += uint64(i)
	}

	squareOfSum = sum * sum

	fmt.Println(squareOfSum - sumOfSquares)
}
